src/js/controllers/confirm.js

```diff
@@ -422,6 +433,47 @@ angular.module('copayApp.controllers').controller('confirmController', function(
       return setSendError(gettextCatalog.getString('No wallet selected'));
     }
 
+    if ($scope.isGlidera) {
+      $scope.get2faCode(function(err, sent) {
+        if (err) {
+          popupService.showAlert(gettextCatalog.getString('Error'), gettextCatalog.getString('Could not send confirmation code to your phone'));
```

Comment:
External services should not be translated. There is a lot of lines using **gettextCatalog** service. Please check and remove all of them.